src/applications/mhv-medical-records/containers/DownloadReportPage.jsx

```diff
@@ -71,6 +76,12 @@ const DownloadReportPage = ({ runningUnitTest }) => {
     false,
   );
 
+  useEffect(() => {
+    return () => {
+      dispatch({ type: Actions.Downloads.BB_CLEAR_ALERT });
+    };
+  }, []);
```

Comment:
Add `dispatch` to the dependency array to get rid of this linting warning.
<img width="600" alt="image" src="https://github.com/user-attachments/assets/94e1482a-78b0-431c-ad91-532186360dc4" />